src/main/scala/com/github/unchama/seichiassist/subsystems/buildcount/subsystems/notification/bukkit/actions/BukkitNotifyBuildAmountThreshold.scala

```diff
@@ -0,0 +1,68 @@
+package com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.bukkit.actions
+
+import cats.Applicative
+import cats.effect.Effect.ops.toAllEffectOps
+import cats.effect.{ConcurrentEffect, IO, SyncIO}
+import com.github.unchama.generic.Diff
+import com.github.unchama.minecraft.actions.OnMinecraftServerThread
+import com.github.unchama.seichiassist.concurrent.PluginExecutionContexts.onMainThread
+import com.github.unchama.seichiassist.subsystems.buildcount.domain.playerdata.BuildAmountData
+import com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.application.actions.NotifyBuildAmountThreshold
+import com.github.unchama.seichiassist.subsystems.discordnotification.DiscordNotificationAPI
+import com.github.unchama.seichiassist.util.{PlayerSendable, SendMessageEffect, SendSoundEffect}
+import org.bukkit.ChatColor.{BOLD, GOLD}
+import org.bukkit.Sound
+import org.bukkit.entity.Player
+
+object BukkitNotifyBuildAmountThreshold {
+
+  import PlayerSendable.forString
+
+  // TODO: BukkitNotifyLevelUpなのにdiffの展開やいつメッセージを出すかなどを扱うべきでない。
+  // see also: https://github.com/GiganticMinecraft/SeichiAssist/blob/8ce9df9e4fc2da8f84d8aed4e2b74bbe95a61a02/src/main/scala/com/github/unchama/seichiassist/subsystems/breakcount/subsystems/notification/bukkit/actions/BukkitNotifyLevelUp.scala
+  def apply[F[_]: OnMinecraftServerThread: ConcurrentEffect: DiscordNotificationAPI]
+    : NotifyBuildAmountThreshold[F, Player] = {
+    new NotifyBuildAmountThreshold[F, Player] {
+      override def ofBuildAmountTo(player: Player)(diff: Diff[BuildAmountData]): F[Unit] = {
+        val Diff(oldBuildAmount, newBuildAmount) = diff
+        val million = 1000000
+        val oldBuildAmountOneMillionUnit = (oldBuildAmount.expAmount.amount / million).toLong
+        val newBuildAmountOneMillionUnit = (newBuildAmount.expAmount.amount / million).toLong
```

Comment:
```suggestion
        val oldBuildAmountOneMillionUnit = oldBuildAmount.expAmount.amount / million
        val newBuildAmountOneMillionUnit = newBuildAmount.expAmount.amount / million
```

現状の総建築量の最大値は約560万程で、取りうる値の範囲が近いうちに約21億を超えるということは考えにくいので`.toLong`せずとも`Int`型で十分です。